### PR TITLE
Updated Logging usage to explain defaultFormatter usage

### PR DESCRIPTION
**Pull request by suls**
Documentation update referencing https://github.com/twitter/util/commit/ece68e5c319d912b469b13bad5cf9490d26dca93

### CODE REVIEW

File: doc/src/sphinx/code/AdvancedServer.scala [hunk 1]
```diff
@@ -18,6 +19,12 @@ object AdvancedServer extends TwitterServer {
   //#stats
   val counter = statsReceiver.counter("requests_counter")
   //#stats
+  //#formatter
+  override def defaultFormatter = new Formatter(
+    timezone = Some("UTC")
+    , prefix = "<yyyy-MM-dd HH:mm:ss.SSS> [%.3s] %s: "
```
Comment (mosesn) [inline doc/src/sphinx/code/AdvancedServer.scala]: can we put the comma on the previous line?

**Comment by mosesn** [comment]
Assuming you tried this out locally, LGTM!

File: doc/src/sphinx/code/AdvancedServer.scala (hunk 1 above)
Comment (suls) [inline doc/src/sphinx/code/AdvancedServer.scala]: :+1:

**Comment by suls** [comment]
By "tried this out locally" do you mean 1/ sbt-site's `previewSite` or 2/ compiled the code?

1/ yes ;)
2/ form here https://github.com/suls/custom-log-for-twitter-server/blob/da540a21d45c9a4d7f07708672535c89daf760a2/src/main/scala/main/Main.scala

**Comment by mosesn** [comment]
Something weird happened . . .

**Comment by suls** [comment]
Sorry for the mess @mosesn - I should remember not to attempt pushing anything in the middle of the night. Being a sleep-deprived parent takes its toll :v:

**Comment by mosesn** [comment]
No problem at all. I'll get one more set of eyes on this and then we'll merge it in.  Thanks!

**Comment by blackicewei** [comment]
LGTM :+1:

**Comment by mosesn** [comment]
Pulling it internally, will keep you posted!

**Comment by mosesn** [comment]
This was merged internally, we'll let you know when it gets pushed to develop.  Thanks again for your contribution!

**Comment by suls** [comment]
:tada: 
thanks for seeing this through! :bow:

**Comment by mosesn** [comment]
Yay!  This hit OSS develop: https://github.com/twitter/twitter-server/commit/447ea4a146fa700ecc8b0998892ba2a9009d67de

**Pull request closed without merging**